tests/gtest/integration_test_population.cpp

```diff
@@ -70,7 +70,7 @@ namespace
                 // and unfished biomass
                 it = input.FindMember("median_R0");
                 rapidjson::Value &R_0 = (*it).value;
-                double rzero = R_0[0].GetDouble();
+                double log_rzero = fims::log(R_0[0].GetDouble());
```

Comment:
Suggest: add comments to explain .GetDouble(). When obtaining the numeric values, GetDouble() will convert internal integer representation to a double. Note that, int and unsigned can be safely converted to double, but int64_t and uint64_t may lose precision (since mantissa of double is only 52-bits).